pkg/resourcemanager/mysql/mysqlaaduser/reconcile.go

```diff
@@ -9,23 +9,24 @@ import (
 	"fmt"
 	"strings"
 
+	mysqlmgmt "github.com/Azure/azure-sdk-for-go/services/mysql/mgmt/2017-12-01/mysql"
+	_ "github.com/go-sql-driver/mysql" //sql drive link
 	"github.com/pkg/errors"
-
-	"github.com/Azure/azure-service-operator/pkg/helpers"
-	"github.com/Azure/azure-service-operator/pkg/resourcemanager/config"
-	"github.com/Azure/azure-service-operator/pkg/resourcemanager/mysql"
-	mysqldatabase "github.com/Azure/azure-service-operator/pkg/resourcemanager/mysql/database"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
 
 	"github.com/Azure/azure-service-operator/api/v1alpha1"
 	"github.com/Azure/azure-service-operator/api/v1alpha2"
 	"github.com/Azure/azure-service-operator/pkg/errhelp"
+	"github.com/Azure/azure-service-operator/pkg/helpers"
 	"github.com/Azure/azure-service-operator/pkg/resourcemanager"
-
-	_ "github.com/go-sql-driver/mysql" //sql drive link
-	"k8s.io/apimachinery/pkg/runtime"
-	"k8s.io/apimachinery/pkg/types"
+	"github.com/Azure/azure-service-operator/pkg/resourcemanager/config"
+	"github.com/Azure/azure-service-operator/pkg/resourcemanager/mysql"
+	mysqlserver "github.com/Azure/azure-service-operator/pkg/resourcemanager/mysql/server"
 )
 
+const mysqlDatabase = "mysql"
```

Comment:
Is it worth a comment here explaining why this magic database name makes sense? (I assume that this is the default DB name at the server level that lets you manage users?)

Reply:
Nice call - I've moved that constant into `pkg/resourcemanager/mysql ` and renamed the server port and driver name constants there so that they don't stutter.